docs/software/gaming.md

```diff
@@ -10,6 +10,7 @@
 - [**DXX-Rebirth - Descent 1 and 2 OpenGL port**](#dxx-rebirth)
 - [**Steam - Steam client**](#steam)
 - [**PaperMC - Fast and optimised Minecraft server**](#papermc)
+- [**Box86 - x86 userspace emulation for ARM**](#box86)
 
```

Comment:
In addition in File software.md (software/#gaming-emulation) the Box86 has to be added:

------<snip>----------
Old:
```
- [**Steam - Steam client**](../software/gaming/#steam)
- [**PaperMC - Fast and optimized Minecraft server**](../software/gaming/#papermc)
```

New:
```
- [**Steam - Steam client**](../software/gaming/#steam)
- [**PaperMC - Fast and optimized Minecraft server**](../software/gaming/#papermc)
- [**Box86 - x86 userspace emulation for ARM**](../software/gaming/#box86)
```